CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Introduce `on_schema_change` config to detect and handle schema changes on incremental models ([#1132](https://github.com/fishtown-analytics/dbt/issues/1132), [#3387](https://github.com/fishtown-analytics/dbt/issues/3387))
 
 ### Fixes
+- Fix config merge behavior with experimental parser [3637](https://github.com/dbt-labs/dbt/pull/3637)
```

Comment:
This should be going into v0.20.1, right?

Reply:
Yes! I'll make a backport PR Monday morning. Hold me to it 😉

Reply:
You got it ;) could you move this changelog entry down below `v0.20.1` before merging?